app/public/index.html

```diff
@@ -30,6 +31,58 @@
   <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
     integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
     crossorigin="anonymous"></script>
+
 </body>
 
+<!-- Footer -->
+<footer class="footer text-center text-lg-start bg-light text-muted orange-background">
+
+  <!-- Section: Links  -->
+  <section class="orange-background">
+    <div class="orange-background text-center text-md-start mt-5">
+      <!-- Grid row -->
+      <div class="row mt-3 orange-background">
+        <!-- Grid column -->
+        <div class="col-md-3 col-lg-4 col-xl-3 mx-auto mb-4 pt-4 orange-background">
+          <!-- Content -->
+          <h6 class="text-uppercase fw-bold mb-6 ">
+            Got It! The recurring to-do list</br>
+          </h6>
+          <p>
+            Developed by Meia Natividad
+            <br>Final Project for Techtonica</br>
+          </p>
+          <p>May 2021 | Oakland, CA
+
+          </p>
+        </div>
+        <!-- Grid column -->
+
+        <!-- Grid column -->
+        <div class="col-md-4 col-lg-3 col-xl-3 mx-auto mb-md-0 mb-4 pt-4 orange-background">
+          <!-- Links -->
+          <h6 class="text-uppercase fw-bold mb-6">
+            Contact
+          </h6>
+          <p class="mb-0">
+            <a href="mailto:meianatividad@gmail.com" target="_blank">Email Me</a>
```

Comment:
For the far future, if you made the app publically indexable on google or other search engines, to not have your email out in public to avoid spam. Maybe have a button that loads your email if it is clicked by a human?

Reply:
Good point! I'll remove that.